core-bundle/tests/EventListener/SubrequestCacheSubscriberTest.php

```diff
@@ -42,14 +42,14 @@ public function testMergesCacheControlHeader(): void
         $this->onKernelRequest($subscriber, KernelInterface::MASTER_REQUEST);
 
         $subResponse = new Response();
-        $subResponse->headers->set(SubrequestCacheSubscriber::MERGE_CACHE_HEADER, true);
+        $subResponse->headers->set(SubrequestCacheSubscriber::MERGE_CACHE_HEADER, '1');
```

Comment:
Again, why `'1'` instead of `1`?

Reply:
see https://github.com/contao/contao/pull/1924#discussion_r452965031